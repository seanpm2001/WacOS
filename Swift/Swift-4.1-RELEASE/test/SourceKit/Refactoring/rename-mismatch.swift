struct Sss {
}

_ = Mismatch()
//  Mismatch()
_ = Sss()

// RUN: rm -rf %t.result && mkdir -p %t.result
// RUN: %sourcekitd-test -req=syntactic-rename -rename-spec %S/syntactic-rename/rename-mismatch.in.json %s >> %t.result/rename-mismatch.expected
// RUN: diff -u %S/syntactic-rename/rename-mismatch.expected %t.result/rename-mismatch.expected

// RUN: rm -rf %t.ranges && mkdir -p %t.ranges
// RUN: %sourcekitd-test -req=find-rename-ranges -rename-spec %S/syntactic-rename/rename-mismatch.in.json %s >> %t.ranges/rename-mismatch.expected
// RUN: diff -u %S/find-rename-ranges/rename-mismatch.expected %t.ranges/rename-mismatch.expected
