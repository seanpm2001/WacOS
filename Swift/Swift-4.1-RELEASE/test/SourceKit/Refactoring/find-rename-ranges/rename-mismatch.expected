source.edit.kind.active:
  1:8-1:11 source.refactoring.range.kind.basename
source.edit.kind.unknown:
source.edit.kind.mismatch:
source.edit.kind.active:
  6:5-6:8 source.refactoring.range.kind.basename
